2022-04-18 09:49:56,496 ERROR 7224 nodejs.ClusterClientNoResponseError: client no response in 170061720ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 7224
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 11:34:18,792 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 425649ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 11:37:28,806 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 150225ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 11:42:54,317 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 305503ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 11:48:32,631 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318216ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 11:55:30,369 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318207ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:01:08,134 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 317754ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:06:27,355 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 204468ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:11:22,563 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 255157ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:17:20,550 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 317973ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:19:43,376 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 122812ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:26:40,999 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318537ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:32:19,811 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318783ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:37:13,287 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 273461ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:43:11,521 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318221ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:48:50,279 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318747ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 12:54:48,845 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318586ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:00:27,527 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318669ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:05:57,507 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 229940ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:11:35,377 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 317923ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:17:34,078 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318693ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:23:12,770 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318681ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:28:47,678 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 318745ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:33:29,248 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 241563ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:38:35,343 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 286081ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:44:37,339 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 321984ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-18 13:48:53,869 ERROR 33947 nodejs.ClusterClientNoResponseError: client no response in 236518ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 33947
hostname: xiongchengdeMacBook-Pro.local

2022-04-19 13:00:15,448 ERROR 1159 nodejs.ClusterClientNoResponseError: client no response in 1901746ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1159
hostname: xiongchengdeMacBook-Pro.local

2022-04-19 13:03:09,792 ERROR 1159 nodejs.ClusterClientNoResponseError: client no response in 174314ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 1159
hostname: xiongchengdeMacBook-Pro.local

2022-04-20 13:00:21,300 ERROR 3702 nodejs.ClusterClientNoResponseError: client no response in 1900185ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 3702
hostname: xiongchengdeMacBook-Pro.local

2022-04-20 13:26:33,000 ERROR 3702 nodejs.ClusterClientNoResponseError: client no response in 1571688ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 3702
hostname: xiongchengdeMacBook-Pro.local

2022-04-20 23:10:30,683 ERROR 3702 nodejs.ClusterClientNoResponseError: client no response in 926871ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/xiongcheng/azure/MyBizIntouch/Intouch-Coach-App/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:436:11)
    at tryOnTimeout (timers.js:300:5)
    at listOnTimeout (timers.js:263:5)
    at Timer.processTimers (timers.js:223:10)
name: "ClusterClientNoResponseError"
pid: 3702
hostname: xiongchengdeMacBook-Pro.local

